FlowCrypt/Functionality/Services/Organisational Rules Service/ClientConfigurationService.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  OrgRulesSignInPersmissionsService.swift
+//  FlowCrypt
+//
+//  Created by Yevhen Kyivskyi on 15.07.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+protocol ClientConfigurationServiceType {
+    func checkForUsingKeyManager() -> String?
+}
+
+class ClientConfigurationService: ClientConfigurationServiceType {
+
+    private let organisationalRulesService: OrganisationalRulesServiceType
+
+    init(organisationalRulesService: OrganisationalRulesServiceType = OrganisationalRulesService()) {
+        self.organisationalRulesService = organisationalRulesService
+    }
+
+    /// - Returns: Error message if not using key manager, returns nil if using key manager
+    func checkForUsingKeyManager() -> String? {
+        let organisationalRules = self.organisationalRulesService.getSavedOrganisationalRulesForCurrentUser()
+        if organisationalRules.isUsingKeyManager {
+            return nil
+        }
```

Comment:
This is already wrong - it's supposed to say `!organisationalRules.isUsingKeyManager`

Have you tested this on your email address on flowcrypt.com domain? It's supposed to work there.

And it's also supposed to continue working on general gmail domains.

Reply:
Explanation: if the user is not using EKM based on the OrgRules, then none of these other checks apply, and therefore we should return.

Reply:
It has a conflict with https://github.com/FlowCrypt/flowcrypt-ios/pull/395/files#r674319141